pylib/anki/collection.py

```diff
@@ -467,6 +481,72 @@ def findDupes(self, fieldName: str, search: str = "") -> List[Tuple[Any, list]]:
     findNotes = find_notes
     findReplace = find_and_replace
 
+    # Search Strings
+    ##########################################################################
+
+    # Helper function for the backend's search string operations.
+    # Pass search strings as 'searches' to normalize.
+    # Pass multiple to concatenate (defaults to 'and').
+    # Pass 'negate=True' to negate the end result.
+    # May raise InvalidInput.
+    def search_string(
+        self,
+        *,
+        negate: bool = False,
+        concat_by_or: bool = False,
+        searches: Optional[List[str]] = None,
+        name: Optional["FilterToSearchIn.NamedFilterValue"] = None,
```

Comment:
I realised, too, that that the name is not that meaningful because in the end, they are all "named" filters. The unique trait of the ones in the enum is that they don't take arguments, but I couldn't come up with a concise name conveying that.
We could actually use them directly in FilterToSearchIn's oneof, I think. But that would mean a whole lot of kwargs more to handle for the frontend. What do you think?